Resources/Prototypes/Maps/arena.yml

```diff
@@ -0,0 +1,47 @@
+- type: gameMap
+  id: Arena
+  mapName: 'Arena'
+  mapPath: /Maps/arena.yml
+  minPlayers: 20
+  stations:
+    arena:
+      mapNameTemplate: '{0} Arena Station {1}'
+      nameGenerator:
+        !type:NanotrasenNameGenerator
+        prefixCreator: 'TG'
+      overflowJobs:
+        - Passenger
+      availableJobs:
+        Passenger: [ -1, -1 ]
+        Bartender: [ 1, 1 ]
+        Botanist: [ 2, 2 ]
+        Chef: [ 2, 2 ]
+        Clown: [ 1, 1 ]
+        Janitor: [ 2, 2 ]
+        Mime: [ 1, 1 ]
+        Captain: [ 1, 1 ]
+        HeadOfPersonnel: [ 1, 1 ]
+        ChiefEngineer: [ 1, 1 ]
+        StationEngineer: [ 2, 3 ]
+        ChiefMedicalOfficer: [ 1, 1 ]
+        MedicalDoctor: [ 3, 5 ]
+        MedicalCyborg: [ 1, 2 ]
+        Mystagogue: [ 1, 1 ]
+        Epistemologist: [ 3, 5 ]
+        HeadOfSecurity: [ 1, 1 ]
+        SecurityOfficer: [ 3, 5 ]
+        Chaplain: [ 1, 1 ]
+        Warden: [ 1, 1 ]
+        Cataloguer: [ 1, 1 ]
+        Lawyer: [ 2, 2 ]
+        Quartermaster: [ 2, 3 ]
+        SalvageTechnician: [ 2, 3 ]
+        Musician: [ 1, 1 ]
+        AtmosphericTechnician: [ 1, 2 ]
+        Gladiator: [ 4, 0 ] # here
```

Comment:
should be 0, 4 (as should any setpreference: false jobs

Reply:
Problem with that is they are set to spawn here 
![image](https://user-images.githubusercontent.com/107660393/201450435-a9d7f993-c2f3-4195-a57d-dc455fd5c9e7.png)

If they late join, they will spawn in the arrivals pods

Reply:
ok

Reply:
Is there a way to ignore the late join spawnpoints?

Reply:
nothing is preventing them from latejoining regardless so that issue exists either way

Reply:
How should I resolve this?

Reply:
just let them latejoin, the technology isn't there yet anyway

maybe i can write job specific latejoin spawns later

Reply:
OK, should I maybe set JoinNotifyCrew =True? 

That's what triggers a notification over the radio, yeah?

Reply:
every latejoin notifies the crew

Reply:
ok